gdk/wizard/tests/test_data.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase
+import pytest
+from gdk.wizard.commands.data import Wizard_data as data
+
+
+def Wizard_dataTest(TestCase):
+    @pytest.fixture(autouse=True)
+    def __inject_fixtures(self, mocker):
+        self.mocker = mocker
+
+    def test_init(self):
+        # self.project_config_file = _get_project_config_file()
+        # self.field_map = get_configuration()
+        # self.project_config = self.field_map["component"]
+        # self.component_name = next(iter(self.project_config))
+        # self.schema = self.get_schema()
+        pass
+
+    def test_get_schema():
+        pass
```

Comment:
I do not understand this. Are you defaulting it to pass?

Reply:
My apologizes, the test cases for this section is not yet complete. I just used pass for now

Reply:
Add a comment then to inform reviewers that it will be updated in future CR.